eZ/Publish/Core/MVC/Symfony/MVCEvents.php

```diff
@@ -85,4 +85,15 @@ final class MVCEvents
      * The event listener method receives a eZ\Publish\Core\MVC\Symfony\Event\ContentCacheClearEvent instance.
      */
     const CACHE_CLEAR_CONTENT = 'ezpublish.cache_clear.content';
+
+    /**
+     * The CONSOLE_INIT event allows you to attach listeners before any command is
+     * loaded by the console. It also allows you react to global arguments before commands are run.
+     *
+     * @Event("eZ\Publish\Core\MVC\Symfony\Event\ConsoleInitEvent")
+     *
+     * @internal For internal use by SiteAccess system on console.
+     * @deprecated Planned for move/contribution to Symfony in the future.
```

Comment:
One more remark here, I'm not sure if having `@deprecated` right now is good. There's no alternative yet and we're not even sure if Symfony would accept such contribution. But that's not a strong requirement from me, just sayin'...